skbio/stats/distance/tests/test_base.py

```diff
@@ -467,6 +468,32 @@ def test_to_data_frame_default_ids(self):
                            index=['0', '1'], columns=['0', '1'])
         assert_data_frame_almost_equal(df, exp)
 
+    def test_to_series_1x1(self):
+        series = self.dm_1x1.to_series()
+        exp = pd.Series([0.0],
+                        pd.MultiIndex(levels=[['a'], ['a']],
+                        labels=[[0], [0]]))
+        exp = exp[exp > 0]
+        assert_series_almost_equal(series, exp)
+
+    def test_to_series_3x3(self):
+        series = self.dm_3x3.to_series()
+        exp = pd.Series([0.0, 0.01, 4.2, 0.01, 0.0, 12.0, 4.2, 12.0, 0.0],
```

Comment:
It's shorter and more readable to define the expected series like this (assuming a 3x3 example matrix):

```python
exp = pd.Series([1.0, 5.0, 42.0], index=[('a', 'b'), ('a', 'c'), ('b', 'c')])
```

Pandas will convert the list of tuples into a `pd.MultiIndex`.